tests/garage/tf/algos/test_batch_polopt2.py

```diff
@@ -18,8 +19,10 @@
 
 class TestBatchPolopt2(TfGraphTestCase):
 
-    @mock.patch.multiple(BatchPolopt2, __abstractmethods__=set())
     # pylint: disable=abstract-class-instantiated, no-member
+    # This test cause low memory with some reason
+    @pytest.mark.flaky
```

Comment:
that would be  really surprising. you seen this in the wild on master?

It seems likely that some other part of the test suite is using all if your memory, and this test just became a victim.

Reply:
that makes sense, I will remove the decorator